server/pulp/server/managers/content/upload.py

```diff
@@ -207,6 +208,13 @@ def import_uploaded_unit(repo_id, unit_type_id, unit_key, unit_metadata, upload_
         try:
             result = importer_instance.upload_unit(transfer_repo, unit_type_id, unit_key,
                                                    unit_metadata, file_path, conduit, call_config)
+            if not result['success_flag']:
```

Comment:
I am not sure if skipping wrong `result` is an option, because we won't be able to tell the user if we succeed in importing unit or not.